Misc/NEWS.d/next/Library/2019-02-12-01-33-08.bpo-35505.N9ba_K.rst

```diff
@@ -0,0 +1,4 @@
+test_imaplib.TestImaplib.test_imap4_host_default_value fails when run on the
+computer with the local IMAP server running. This may happen for developers,
+who consequently don't need to be bothered about this test failing when
+working on other CPython patches.
```

Comment:
Hum, I suggest a shorter description: "Fix test_imaplib when a local IMAP server is running."

Reply:
Like this?